goldenmaster/Plugins/ApiGear/Source/ApiGearOLink/Public/UnrealOLinkHost.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "Subsystems/GameInstanceSubsystem.h"
+#include "Templates/PimplPtr.h"
+THIRD_PARTY_INCLUDES_START
+#include "olink/remotenode.h"
+#include "olink/remoteregistry.h"
+THIRD_PARTY_INCLUDES_END
+#include "unrealolinkhost.generated.h"
+
+class UnrealOLinkHostWorker;
+
+/// @brief the public wrapper for the OLink server
```

Comment:
I am trying not to expose the registry to the actual user since it is more confusing than really helpful.
in UE host = registry. Want another registry? created a new host on a different port